hawkmoth/docstring.py

```diff
@@ -130,19 +130,22 @@ def _nest(text, nest):
         Returns:
             str: Indented reST documentation string.
         """
-        return re.sub('(?m)^(?!$)', '   ' * nest, text)
+        lines[:] = [re.sub('^(?!$)', '   ' * nest, line) for line in lines]
```

Comment:
Sorry I missed it before, but the docstring of the function mentions a return value.

Reply:
Force pushed with the fix.

For a code documentation tool we have pretty lousy code documentation ourselves. :D